bundles/org.openhab.binding.energidataservice/README.md

```diff
@@ -69,6 +71,41 @@ This has the following advantages:
 If you want electricity tax included in your total price, please add either `electricity-tax` or `reduced-electricity-tax` to the group - depending on which one applies.
 See [Electricity Tax](#electricity-tax) for further information.
 
+##### Time Series
+
+Group items with aggregate functions are not automatically recalculated into the future when time series for child items are updated.
+Therefore, the `SUM` function mentioned above will only work for the current price.
+Calculation of future total prices can be achieved with a rule:
+
+:::: tabs
+
+::: tab JavaScript
+
```

Comment:
Probably also mention that the example is for a file-based rule.

Reply:
Done.